### PR TITLE
Fix simulcast factory not sending back EncoderInfo

### PR DESCRIPTION
**Pull request by davidliu**

### CODE REVIEW

File: livekit-android-sdk/src/main/java/io/livekit/android/webrtc/SimulcastVideoEncoderFactoryWrapper.kt [hunk 1]
```diff
@@ -165,6 +168,31 @@ internal class SimulcastVideoEncoderFactoryWrapper(
...
+            return future.get()
+        }
+
+        override fun getResolutionBitrateLimits(): Array<VideoEncoder.ResolutionBitrateLimits> {
+            val future = executor.submit(Callable { return@Callable encoder.resolutionBitrateLimits })
+            return future.get()
+        }
+
+        override fun getEncoderInfo(): VideoEncoder.EncoderInfo {
+            val future = executor.submit(Callable { return@Callable encoder.encoderInfo })
```
Comment (davidliu) [inline livekit-android-sdk/src/main/java/io/livekit/android/webrtc/SimulcastVideoEncoderFactoryWrapper.kt:193]: This is the main thing that was breaking the hardware encoder from working (and preventing H264 from working). EncoderInfo holds onto information needed for resolution alignment, which is used to adjust the resolution (only works on 16x16 aligned frames).

**Review by hiroshihorie — APPROVED** [review]
Nice fix !

**Comment by davidzhao** [comment]
would flutter have the same issue then?

**Comment by davidliu** [comment]
Very likely, I'll put in a PR for Flutter tomorrow.

It looks like this behavior got broken in the M97 upgrade, so the device farm testing will hopefully help catch these breakages down the road.

**Pull request merged**